ontology/uco/core/core.ttl

```diff
@@ -598,6 +598,13 @@ core:externalReference
 	rdfs:range core:ExternalReference ;
 	.
 
+core:hasCharacterization
```

Comment:
This property had no accompanying namespace suggestion.  Should this go into `core:`?  I'm guessing so for now, because of the appearances in the Environment and Tool namespaces.
   - What is its domain?
   - What is its range - anything aside from UcoObject, at the most-general property level?  Why did the definition string you provided say "references the Software or Device that make up the actual tool," when the suggested constraints in property shapes were much more general?